src/App.tsx

```diff
@@ -1,12 +1,17 @@
 import 'react-native-gesture-handler'
-import React from 'react'
+import React, { useEffect } from 'react'
 import { SafeAreaProvider } from 'react-native-safe-area-context'
 import { ThemeProvider } from '@shopify/restyle'
 import { Platform, StatusBar } from 'react-native'
+import OneSignal from 'react-native-onesignal'
 import { theme } from './theme/theme'
 import NavigationRoot from './navigation/NavigationRoot'
 
 const App = () => {
+  useEffect(() => {
+    OneSignal.setAppId('64cb14bf-7121-4f41-997f-25e1917e6a1f')
```

Comment:
these types of ids aren't intended to be secure, but is there any standard way to inject them as env vars at build time?

Reply:
yes, thanks for reminding me. I pulled that over from the other app and forgot to come back to it.